### PR TITLE
[15.0][FIX]account_commission: Settlement on reversed moves

### PR DESCRIPTION
**Pull request by Tisho99**
There is an error on a use case generating the settlements for the commission on invoices. Steps to reproduce

1 Set up a commission with Invoice State based on paids
2 Create an invoice, confirm it, and asociate his lines to an agent.
3 Generate a credit note to make a full refund of the previous invoice.

4 Settle the commissions for the selected agent. The reversed invoice is not settled and it musts. 

The addition of a unit test is pending

### CODE REVIEW

**Comment by Tisho99** [comment]
The PR is ready @HaraldPanten @ValentinVinagre

File: account_commission/tests/test_account_commission.py [hunk 1]
```diff
@@ -497,3 +497,32 @@ def test_multi_currency(self):
...
+        # Create an invoice and refund it
+        invoice = self._create_invoice(agent, commission, today, currency=None)
+        invoice.action_post()
+        move_reversal = (
+            self.env["account.move.reversal"]
+            .with_context(active_model="account.move", active_ids=invoice.id)
+            .create(
+                {
+                    "reason": "no reason",
+                    "refund_method": "cancel",
```
Comment (ValentinVinagre) [inline account_commission/tests/test_account_commission.py:572]: Perform the tests with the 3 options

**Review by ValentinVinagre — CHANGES_REQUESTED** [review]

File: account_commission/tests/test_account_commission.py (hunk 1 above)
Comment (Tisho99) [reply account_commission/tests/test_account_commission.py:572]: Done!

**Review by ValentinVinagre — APPROVED** [review]
LGTM 👍🏻

**Comment by ValentinVinagre** [comment]
@HaraldPanten @Jesarregui , could you review it please?

**Review by Jesarregui — APPROVED** [review]
LGTM

**Comment by ValentinVinagre** [comment]
@pedrobaeza do you see it correct?

File: account_commission/readme/CONFIGURE.rst [hunk 2]
```diff
@@ -3,4 +3,4 @@ For selecting invoice status in commissions:
 #. Edit or create a new record to select the invoice status for settling the commissions.
 
    * **Invoice Based**: Commissions are settled when the invoice is issued.
-   * **Payment Based**: Commissions are settled when the invoice is paid.
+   * **Payment Based**: Commissions are settled when the invoice is paid (or refunded).
```
Comment (pedrobaeza) [inline account_commission/readme/CONFIGURE.rst]: I think this requires a bit more of explanation:

```suggestion
   * **Payment Based**: Commissions are settled when the invoice is paid or refunded. Note that when refunding an invoice, the corresponding reversed commission will be settled as well, resulting in a 0 net commission between both operations.
```

Comment (Tisho99) [reply account_commission/readme/CONFIGURE.rst]: Done! The PR is ready.

Thanks for the review

**Comment by pedrobaeza** [comment]
/ocabot merge patch

Please forward-port it to v16

**Comment by ValentinVinagre** [comment]
@Tisho99 Can you also make the change in v.16?

**Pull request merged**